Assets/MixedRealityToolkit/Sharing/MixedReality.Toolkit.Sharing.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
Let's remove this file. The core toolkit will be distributed as a single package.